src/VoteMonitor.Api.DataExport/Handlers/DataExportQueryHandler.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using VoteMonitor.Api.DataExport.Model;
+using VoteMonitor.Api.DataExport.Queries;
+using VoteMonitor.Entities;
+
+namespace VoteMonitor.Api.DataExport.Handlers
+{
+    public class DataExportQueryHandler : IRequestHandler<GetDataForExport, List<ExportModel>>
+    {
+        private readonly VoteMonitorContext _context;
+        private readonly ILogger _logger;
+
+        public DataExportQueryHandler(VoteMonitorContext context, ILogger logger)
+        {
+            _context = context;
+            _logger = logger;
+        }
+
+        public async Task<List<ExportModel>> Handle(GetDataForExport request, CancellationToken cancellationToken)
+        {
+            var exportData = await _context.Answers
+                  .Where(a => a.IdObserver > 10)
```

Comment:
Usually we make a filtering for Ids that belong to internal testers, and they usually are dependent on the saved internal testers accounts.
This value is dynamic and it should be a parameter.